projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-site-logo-endpoint.php

```diff
@@ -68,29 +66,29 @@ function callback( $path = '', $site_id = 0 ) {
 
 		$args = $this->input();
 		$logo_settings = $this->get_current_settings();
+
 		if ( empty( $args ) || ! is_array( $args ) ) {
 			return $logo_settings;
 		}
 
 		if ( isset( $args['id'] ) ) {
-			$logo_settings['id'] = (int) $args['id'];
-		}
-		if ( isset( $args['url'] ) ) {
```

Comment:
Is it worth returning an error if this endpoint is called with the `url` argument? Or perhaps some kind of deprecation warning?

Reply:
I can't find stats, but I have a sense this endpoint isn't even used any longer--because the site_logo feature only works with legacy premium themes, any client that wants to update the logo would likely use another method. It doesn't seem to be used by Calypso at all.

Reply:
Note that I also thought about marking these endpoints as deprecated, but since they're undocumented that doesn't really do anything--all `'deprecated' => true` does is update the WP.com API documentation site.

Reply:
Yes, I can't find any stats either. The ones I can find related to site logos have 4 requests in the last year, so this doesn't seem to be active anymore